udphook.go

```diff
@@ -19,6 +19,21 @@ const (
 // UDPOptionSetter represents functional arguments passed to ELKHook
 type UDPOptionSetter func(f *UDPHook)
 
+// ClientNet sets clientNet option in UDPHook
+// Usage: hook, err := NewUDPHook(ClientNet("tcp"))
+func ClientNet(clientNet string) UDPOptionSetter  {
+	return func(f *UDPHook) {
+		f.clientNet = clientNet
+	}
+}
+// ClientAddr sets clientAddr option in UDPHook
+// Usage: hook, err := NewUDPHook(ClientAddr("192.168.1.1:65000"))
+func ClientAddr(clientAddr string) UDPOptionSetter  {
```

Comment:
overall makes sense, one request though - instead of two functional argument parameters, let's make it one:

```go
func Target(network, addr string) UDPOptionSetter {
    return func(f *UDPHook) {
          f.targetNet = network
          f.targetAddr = addr
    }
}
```

Reply:
Yes, that makes sense, simplifying calls. I can add it. But I think what if somebody needs to set only one option? Or later we need to add 3rd option? I guess setting options one-by-one forces more writing, but makes things more flexible. One can set options only he/she needs and in any order. What do you think of it?

Reply:
we can address those problems later, I don't think this is a big deal for now

Reply:
I've added `Target` to set them at one call.